external/ilasm/ilasm.depproj

```diff
@@ -6,6 +6,22 @@
     <NugetRuntimeIdentifier>$(RuntimeOS)-x64</NugetRuntimeIdentifier>
     <OutputPath>$(ToolsDir)ilasm</OutputPath>
     <IsRuntimeAssembly>false</IsRuntimeAssembly>
+    <RidSpecificAssets>true</RidSpecificAssets>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NETCore.Platforms">
+      <Version>2.0.0-$(CoreFxExpectedPrerelease)</Version>
```

Comment:
Agreed, that would be better. I'm not very familiar with the format of dependencies.props, but if we can get that to work then I'm all for it.